.actions/setup_tools.py

```diff
@@ -236,6 +236,17 @@ def create_mirror_package(src_folder: str, lit_pkg_mapping: dict) -> None:
         copy_adjusted_modules(src_folder, pkg_name, lit_name, lit_pkg_mapping)
 
 
+def distribute_version(src_folder: str, ver_file: str = "version.info") -> None:
```

Comment:
Do we still need this? The `__version__` files will read the info file contents, no need to rewrite

Reply:
the story is about having version written only in single file not as it is now in 4 different

Reply:
because that is not part of any (standalone) package :otter:

Reply:
Since it's in the `MANIFEST.in` (https://github.com/Lightning-AI/lightning/pull/15374/files#diff-41d5a52589e0480be9c099d2bba7a8135b8b0d71bcbb8df3582a8df1c2295003R5), the standalone packages should have access to it, then we don't need one per project.

Reply:
Maybe I should try to make these changes?

Reply:
This is explained in point (4) of https://packaging.python.org/en/latest/guides/single-sourcing-package-version/

Reply:
Yes, but if it is not in project it can be overwritten by any package or user...